x-pack/plugins/apm/scripts/upload-telemetry-data/index.ts

```diff
@@ -49,8 +49,9 @@ async function uploadData() {
     ...(httpAuth
       ? {
           auth: { ...httpAuth, username: 'elastic' },
+          ssl: { rejectUnauthorized: false },
         }
-      : {}),
+      : { ssl: { rejectUnauthorized: false } }),
```

Comment:
Needed to add these to run against the observability cluster. Not proud of it but it's for running against a known cluster for testing purposes so not going to worry too much about lax SSL.

Reply:
IIRC (big if), you can also set NODE_TLS_REJECT_UNAUTHORIZED

Reply:
It works. I'll take this part out and add that to the docs.